src/features/advanced/cxpo-auto-pricing.ts

```
@@ -0,0 +1,200 @@
+  return { buyValue: buyValue, sellValue: sellValue, showBuy: showBuy, showSell: showSell };
+}
+
+function getInfoFromOrderType(orders: PrunApi.CXBrokerOrder[], quantityNeeded: number) {
+  let quantityCounted = 0;
+  let volumeAccum = 0;
+  for (let i = 0; i < orders.length; i++) {
+    const offerAmt = orders[i].amount;
+    if (!offerAmt) {
+      return [0, 0, 0];
+    }
+    const offerPrice = orders[i].limit!.amount;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)